server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 5000;
+
+// console.log that your server is up and running
+app.listen(port, () => console.log(`Listening on port ${port}`));
```

Comment:
I know it seems silly, but typically this line is at the bottom of the file. You should define the routes before you start the server.

Reply:
Moved line 6 to the bottom of file.